README.md

```diff
@@ -174,6 +174,8 @@ At last, there are a lot of functionalities [not implemented yet](https://github
 
 # License
 
+This project started as a fork of [SQLDroid](https://github.com/SQLDroid/SQLDroid) and then it was rewritten in Scala.
+
 The MIT License (MIT)
 
 Copyright (C) 2012 47 Degrees, LLC http://47deg.com hello@47deg.com
```

Comment:
This is the date of the original project
On Nov 27, 2015 14:30, "Francisco Pérez Paradas" <notifications@github.com>
wrote:

> In README.md
> <https://github.com/47deg/mvessel/pull/38#discussion_r46046797>:
>
> > @@ -174,6 +174,8 @@ At last, there are a lot of functionalities [not implemented yet](https://github
> >
> >  # License
> >
> > +This project started as a fork of [SQLDroid](https://github.com/SQLDroid/SQLDroid) and then it was rewritten in Scala.
> > +
> >  The MIT License (MIT)
> >
> >  Copyright (C) 2012 47 Degrees, LLC http://47deg.com hello@47deg.com
>
> Copyright 2012?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/47deg/mvessel/pull/38/files#r46046797>.
>